### DISCUSSION TITLE
How to obtain the MAC address for BLE use?

### DISCUSSION THREAD

**Discussion by MikeHotel0815** [syssi/esphome-jk-bms#143] [Q&A] [answered]
Hi,
how do I obtain the MAC address of an JK BMS? 

thanx in advance

MikeHotel

**Comment by syssi** [comment, marked as answer]
Just flash the BLE yaml with a random MAC and let the ESP boot. You will see all discovered/available BLE at the log. Pick the one with a `JK-...` name.

**Reply by MikeHotel0815** [reply]
Thank you! Great work btw.
